packages/varnish/src/theme/theme.ts

```diff
@@ -60,63 +83,120 @@ export const getVarnishDesignTokens = (): ThemeOptions => {
           }),
         },
       },
-      MuiAppBar: {
-        styleOverrides: {
-          root: ({ ownerState, theme }) => ({
-            ...(ownerState.color === undefined || ownerState.color === 'default'
-              ? {
-                  color: theme.palette.text.primary,
-                  background: color2.N1.hex,
-                }
-              : {}),
-          }),
-        },
-      },
       MuiRating: {
         styleOverrides: {
           root: {
             color: color2.B3.hex,
           },
         },
       },
-      MuiSwitch: {
-        styleOverrides: {
-          thumb: ({ ownerState }) => ({
-            ...(ownerState.color === undefined || ownerState.color === 'default'
-              ? {
-                  backgroundColor: color2.B3.hex,
-                }
-              : {}),
-          }),
-          track: ({ ownerState }) => ({
-            ...(ownerState.color === undefined || ownerState.color === 'default'
-              ? {
-                  backgroundColor: color2.A3.hex,
-                }
-              : {}),
-          }),
-        },
-      },
     },
     typography: {
-      fontFamily: [
-        '"Lato"',
-        '"Lucida Grande"',
-        '"Tahoma"',
-        '"Helvetica Neue"',
-        '"Helvetica"',
-        '"-apple-system"',
-        '"BlinkMacSystemFont"',
-        '"Segoe UI"',
-        '"PingFang SC"',
-        '"Hiragino Sans GB"',
-        '"Microsoft YaHei"',
-        '"Arial"',
-        '"sans-serif"',
-        '"Apple Color Emoji"',
-        '"Segoe UI Emoji"',
-        '"Segoe UI Symbol"',
-      ].join(','),
+      fontFamily,
+      fontWeightLight, // 300
```

Comment:
nit: do we want to preserve these comments here in case these values change up top in the future? might get hard to maintain consistency across these comments unknowingly

Reply:
yes, ill remove next them i touch the code.. or you can